tests/integration_tests/modules/test_ubuntu_advantage.py

```diff
@@ -107,6 +115,25 @@ def test_valid_token(self, client: IntegrationInstance):
         assert status.ok
         assert "This machine is not attached" not in status.stdout
 
+    @pytest.mark.user_data(ATTACH.format(token=CLOUD_INIT_UA_TOKEN))
+    def test_idempotency(self, client: IntegrationInstance):
+        assert CLOUD_INIT_UA_TOKEN, "CLOUD_INIT_UA_TOKEN env var not provided"
+        log = client.read_from_file("/var/log/cloud-init.log")
+        verify_clean_log(log)
+        status = client.execute("ua status")
+        assert status.ok
+        assert "This machine is not attached" not in status.stdout
```

Comment:
Just to avoid fragility in the event of internationalization of messages at some point. Let's use `ua status --format json` here and look at `{"enabled": True|False}` in the response in this case and below

Reply:
done